kernel/src/mm/phys/mod.rs

```diff
@@ -102,6 +134,8 @@ fn book_mmio_regions(
                 Ok(()) => {},
                 // Frame lies outside addressable physical memory.
                 Err(e) if e.code == ErrorCode::InvalidArgument => {},
+                // Frame already allocated, continue try booking.
+                Err(e) if e.code == ErrorCode::ResourceBusy => {},
```

Comment:
If this happens is actually an error. It means that another region has booked it and we are just trying to supress the error.